docs/content/introduction.md

```diff
@@ -128,28 +125,27 @@ My MTA will thus have to support two kinds of Submission:
 
 ```txt
  ┏━━━━ Outward Submission ━━━━┓
+
                     ┌────────────────────┐                    ┌┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┐
 Me ---------------> ┤                    ├ -----------------> ┊                 ┊
                     │       My MTA       │                    ┊ Third-party MTA ┊
                     │                    ├ <----------------- ┊                 ┊
                     └────────────────────┘                    └┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┘
+
                                ┗━━━━━━━━━━ Inward Submission ━━━━━━━━━━┛
 ```
 
-##### Outward Submission
-
-The best practice as of 2020 when it comes to securing Outward Submission is to use _Implicit TLS connection via ESMTP on port 465_ (see [RFC 8314][rfc-8314]). Let's break it down.
+#### Outward Submission
 
-- Implicit TLS means the server _enforces_ the client into using an encrypted TCP connection, using [TLS][wikipedia-tls]. With this kind of connection, the MUA _has_ to establish a TLS-encrypted connection from the get go (TLS is implied, hence the name "Implicit"). Any client attempting to either submit email in cleartext (unencrypted, not secure), or requesting a cleartext connection to be upgraded to a TLS-encrypted one using `STARTTLS`, is to be denied. Implicit TLS is sometimes called Enforced TLS for that reason.
-- [ESMTP][wikipedia-esmtp] is [SMTP][wikipedia-smtp] + extensions. It's the version of the SMTP protocol that a mail-server commonly communicates with today. For the purpose of this documentation, ESMTP and SMTP are synonymous.
-- Port 465 is the reserved TCP port for Implicit TLS Submission (since 2018). There is actually a boisterous history to that ports usage, but let's keep it simple.
+The best practice as of 2020 when it comes to securing Outward Submission is to use _Implicit TLS connection via ESMTP on port 465_ (see [RFC 8314][rfc-8314]). Please read our article about [**Understanding the Ports**][docs-understandports] for more details!
```

Comment:
Although we are encouraging this, DMS doesn't actually support it well right now :sweat_smile: 

I'd need to look over this section more thoroughly, but AFAIK it's only applicable for sending to a relay / smart host, I'm not sure if this article clarified that. Users should understand that it's otherwise port 25, and this is more about choosing between port 587 or 465 for the most part (_or alternative submission ports offering explicit / implicit TLS_).

```suggestion
When it comes to securing Outward Submission you should prefer to use _Implicit TLS connection via ESMTP on port 465_ (see [RFC 8314][rfc-8314]). Please read our article about [**Understanding the Ports**][docs-understandports] for more details!
```

Reply:
👍🏼